src/components/DatePicker.ts

```diff
@@ -42,7 +41,7 @@ export class DatePicker extends Component<DatePickerProps, DatePickerState> {
             printDate: `12/05/1994`
         };
         this.handleClick = this.handleClick.bind(this);
-        this.handleChange = this.handleChange.bind(this);
+        // this.handleChange = this.handleChange.bind(this);
```

Comment:
Remove if not needed `// this.handleChange = this.handleChange.bind(this);`

Reply:
Done

Reply:
Done